packages/server/database/types/OrganizationUser.ts

```diff
@@ -26,7 +24,7 @@ export default class OrganizationUser {
 
   constructor(input: Input) {
     const {userId, id, removedAt, inactive, orgId, joinedAt, newUserUntil, role, tier} = input
-    this.id = id || generateUID()
+    this.id = id || `${userId}::${orgId}`
```

Comment:
we can't do this because we need to keep track of people who were in an org, left, and then came back